Userland/Utilities/hexdump.cpp

```diff
@@ -68,13 +71,29 @@ ErrorOr<int> serenity_main(Main::Arguments args)
     Span<u8> previous_line;
     static_assert(LINE_LENGTH_BYTES * 2 <= contents.size(), "Buffer is too small?!");
     size_t contents_size = 0;
+    size_t bytes_read = 0;
+    size_t bytes_remaining = 0;
+    int bytes_to_read = 0;
```

Comment:
Is there any reason `bytes_to_read` can't also be a `size_t`? It would avoid the casts on lines 86 and 87.

Reply:
I originally made it an int to satisfy the type signature of `file->read()`, but I've replaced it with size_t to simplify the casts.